program/src/instructions/take_bid_common.rs

```diff
@@ -95,26 +96,26 @@ pub fn take_bid_shared(args: TakeBidArgs) -> Result<()> {
             currency,
             asset_id: Some(asset_id),
         }),
-        tcomp_prog,
+        marketplace_prog,
         TcompSigner::Bid(bid_state),
     )?;
 
     // --------------------------------------- sol transfers
 
     //if margin is used, move money into bid first
     if let Some(margin_pubkey) = bid_state.margin {
-        let decoded_margin_account = assert_decode_margin_account(margin, owner)?;
+        let decoded_margin_account = assert_decode_margin_account(escrow, owner)?;
```

Comment:
Why not be consistent and call this `decoded_escrow_account` and import `assert_decode_margin_account as assert_decode_escrow_account`?

Reply:
also fix comment `margin` --> `escrow` and `Some(margin_pubkey)` -> `Some(escrow_pubkey)`. I think the only thing we can't change is the actual `margin` field on `BidState`, right?

Reply:
Yes can't change the field name (we could, but will break clients... which we already do).